lib/routes/account.js

```diff
@@ -929,7 +929,11 @@ module.exports = (
             && ! doSigninConfirmation
             && emailRecord.emailVerified
           if (shouldSendNewDeviceLoginEmail) {
-            const secondaryEmails = features.isSecondaryEmailEnabled(emailRecord.email) ? db.accountEmails(sessionToken.uid) : P.resolve([])
+            const secondaryEmails = features.isSecondaryEmailEnabled(emailRecord.email) ? db.accountEmails(sessionToken.uid) : P.resolve([{
```

Comment:
I am not sure if we need `features.isSecondaryEmailEnabled` check here. All mailer functions accept an array of emails as first parameter.